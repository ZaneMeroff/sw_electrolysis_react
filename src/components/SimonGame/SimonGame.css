.simon-game-outer-container {
  align-items: center;
  background-color: #313131;
  display: flex;
  flex-direction: column;
  height: 500px;
  padding-bottom: 100px;
  text-align: center;
  width: 100%;
}

.simon-title {
  color: #FFF;
  font-family: 'Press Start 2P', cursive;
  font-size: 150%;
  margin-bottom: 20px;
  margin-top: 60px;
}

.simon-clip-path-bottom {
  background-color: #313131;
  clip-path: polygon(0% 0%, 100% -50%, 100% 100%);
  height: 80px;
  margin-bottom: -80px;
  width: 100%;
}

@media only screen and (max-width: 1000px) {
  
  .simon-clip-path-bottom {
    height: 60px;
    margin-bottom: -60px;
  }

}

@media only screen and (max-width: 500px) {

  .simon-clip-path-bottom {
    height: 40px;
    margin-bottom: -40px;
  }

  .simon-game-outer-container {
    padding-bottom: 80px;
  }

}
