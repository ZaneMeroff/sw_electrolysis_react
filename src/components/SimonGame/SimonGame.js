import React from 'react';
import { GameBoard } from '../GameBoard/GameBoard';
import './SimonGame.css';

export const SimonGame = () => {
  return (
    <div>
      <section id='simon-section' className='simon-game-outer-container'>
        <h2 className='simon-title'>Mr. Simon Game</h2>
        <GameBoard />
      </section>
      <div className="simon-clip-path-bottom"></div>
    </div>
  )
}
