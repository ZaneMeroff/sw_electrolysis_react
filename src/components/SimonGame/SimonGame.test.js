import React from 'react';
import { SimonGame } from './SimonGame';
import { shallow } from 'enzyme';

describe('SimonGame', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<SimonGame />);
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});
