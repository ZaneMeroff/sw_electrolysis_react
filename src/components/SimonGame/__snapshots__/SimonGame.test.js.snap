// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimonGame should match the snapshot 1`] = `
<section
  className="simon-game-outer-container"
  id="simon-section"
>
  <h2
    className="simon-title"
  >
    Mr. Simon Game
  </h2>
  <GameBoard />
</section>
`;
